src/System.Management.Automation/help/HelpUtils.cs

```diff
@@ -12,21 +12,21 @@ internal class HelpUtils
     {
         private static string userHomeHelpPath = null;
 
-
         /// <summary>
         /// Get the path to $HOME
         /// </summary>
         internal static string GetUserHomeHelpSearchPath()
         {
             if (userHomeHelpPath == null)
             {
+                string psVersion = PSVersionInfo.PSVersion.ToString();
 #if UNIX
                 var userModuleFolder = Platform.SelectProductNameForDirectory(Platform.XDG_Type.USER_MODULES);
                 string userScopeRootPath = System.IO.Path.GetDirectoryName(userModuleFolder);
 #else
                 string userScopeRootPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments), "PowerShell");
 #endif
-                userHomeHelpPath = Path.Combine(userScopeRootPath, "Help");
+                userHomeHelpPath = Path.Combine(userScopeRootPath, "Help", psVersion);
```

Comment:
The name may be too specific, so my 6.0.3 PSCore cannot use the existing help content I downloaded with 6.0.1 ...